plugin-server/src/types.ts

```diff
@@ -236,6 +236,7 @@ export enum JobName {
 export type PluginId = Plugin['id']
 export type PluginConfigId = PluginConfig['id']
 export type TeamId = Team['id']
+export type OrgId = Team['organization_id']
```

Comment:
```suggestion
export type OrganizationId = Team['organization_id']
```

Everything else is fully typed out - let's be consistent.